EventTriggeredCalcTests/UnitTests.cs

```diff
@@ -185,58 +178,50 @@ public void EventTriggeredCalcTest()
                 Thread.Sleep(2000);
 
                 // Write to the non-triggering input tag. This way if it triggers the calculation, the timestamps will be off by one, failing the test later.
-                foreach (var context in contextElementList)
+                foreach (AFElement context in contextElementList)
                 {
-                    context.Attributes["Volume"].Data.UpdateValue(new AFValue(volValue, DateTime.Now), AFUpdateOption.Replace);
+                    context.Attributes["Volume"].Data.UpdateValue(new AFValue(VolValue, DateTime.Now), AFUpdateOption.Replace);
                 }
 
                 // Pause to give the calculations enough time to complete
                 Thread.Sleep(settings.UpdateCheckIntervalMS * 2);
 
                 // Cancel the operation and wait for the sample to clean up
                 source.Cancel();
-                var outcome = success.Result;
+                bool outcome = success.Result;
 
                 // Dispose of the cancellation token source
-                if (source != null)
-                {
-                    Console.WriteLine("Disposing cancellation token source...");
-                    source.Dispose();
-                }
+                _testOutputHelper.WriteLine("Disposing cancellation token source...");
+                source.Dispose();
 
                 // Confirm that the sample ran cleanly
                 Assert.True(success.Result);
                 #endregion // step4
 
                 #region step5
-                Console.WriteLine("TEST: Confirming values written at the correct times...");
+                _testOutputHelper.WriteLine("TEST: Confirming values written at the correct times...");
 
-                foreach (var context in contextElementList)
+                foreach (AFValues afValues in contextElementList.Select(context => context.Attributes["Moles"].Data.RecordedValuesByCount(DateTime.Now, TotalExpectedValuesWritten + 2, false, AFBoundaryType.Inside, myPISystem.UOMDatabase.UOMs["mol"], null, false)))
```

Comment:
I'm curious about peoples' thoughts on this change :)

Reply:
I think it is clear what is going on, especially since it is in the unit tests